fluss-server/src/test/java/com/alibaba/fluss/server/replica/AdjustIsrITCase.java

```diff
@@ -63,7 +73,7 @@ void beforeEach() {
     }
 
     @Test
-    void testIsrShrinkAndExpend() throws Exception {
+    void testIsrShrinkAndExpand() throws Exception {
```

Comment:
Add some comment on this test method that this method tests Isr change of log lag.